public/data/python.json

```diff
@@ -239,6 +578,131 @@
         ],
         "tags": ["python", "math", "prime", "check"],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Generate Fibonacci Sequence",
+        "description": "Generates a Fibonacci sequence up to a given number of terms.",
+        "code": [
+          "def generate_fibonacci(n):",
```

Comment:
Could you maybe use a yield inside this snippet ? It would better show how to calculate one number, while supporting  keeping the support of calculating multiple